### ISSUE TITLE
[Bug] BE crash when using Stream load 

### ISSUE THREAD

**Issue by zh0122** [apache/incubator-doris#5246]
**Describe the bug**
BE crash when using Stream load sometimes.

Most of the Flink jobs works well.

**To Reproduce**

Stream Load over Flink with null value

**Expected behavior**

Stream load successfully

**Log**

> There is no log in be.INFO due to the BE crash.

- be.out
```log
start time: Wed Jan 13 14:59:29 CST 2021
*** Aborted at 1610521298 (unix time) try "date -d @1610521298" if you are using GNU date ***
PC: @           0xda5e87 rapidjson::GenericValue<>::Accept<>()
*** SIGSEGV (@0xe) received by PID 13324 (TID 0x7f2c78721700) from PID 14; stack trace: ***
    @     0x7f2d68309280 (unknown)
    @           0xda5e87 rapidjson::GenericValue<>::Accept<>()
    @          0x15fc469 _ZN5doris10JsonReader17_print_json_valueB5cxx11ERKN9rapidjson12GenericValueINS1_4UTF8IcEENS1_19MemoryPoolAllocatorINS1_12CrtAllocatorEEEEE
    @          0x15fd1d9 doris::JsonReader::_write_values_by_jsonpath()
    @          0x15fef2f doris::JsonReader::_handle_flat_array_complex_json()
    @          0x15ff9a1 doris::JsonScanner::get_next()
    @          0x15b6254 doris::BrokerScanNode::scanner_scan()
    @          0x15b6cf1 doris::BrokerScanNode::scanner_worker()
    @          0x26cfccf execute_native_thread_routine
    @     0x7f2d680bedd5 start_thread
    @     0x7f2d683d0ead __clone
start time: Wed Jan 13 15:01:46 CST 2021
*** Aborted at 1610521344 (unix time) try "date -d @1610521344" if you are using GNU date ***
PC: @           0xda5e87 rapidjson::GenericValue<>::Accept<>()
*** SIGSEGV (@0xe) received by PID 17474 (TID 0x7fed64b50700) from PID 14; stack trace: ***
    @     0x7fee3b31d280 (unknown)
    @           0xda5e87 rapidjson::GenericValue<>::Accept<>()
    @          0x15fc469 _ZN5doris10JsonReader17_print_json_valueB5cxx11ERKN9rapidjson12GenericValueINS1_4UTF8IcEENS1_19MemoryPoolAllocatorINS1_12CrtAllocatorEEEEE
    @          0x15fd1d9 doris::JsonReader::_write_values_by_jsonpath()
    @          0x15fef2f doris::JsonReader::_handle_flat_array_complex_json()
    @          0x15ff9a1 doris::JsonScanner::get_next()
    @          0x15b6254 doris::BrokerScanNode::scanner_scan()
    @          0x15b6cf1 doris::BrokerScanNode::scanner_worker()
    @          0x26cfccf execute_native_thread_routine
    @     0x7fee3b0d2dd5 start_thread
    @     0x7fee3b3e4ead __clone
start time: Wed Jan 13 15:02:28 CST 2021
start time: Thu Jan 14 16:54:42 CST 2021
*** Aborted at 1610617561 (unix time) try "date -d @1610617561" if you are using GNU date ***
PC: @           0xda5e87 rapidjson::GenericValue<>::Accept<>()
*** SIGSEGV (@0xe) received by PID 22183 (TID 0x7fed14ef8700) from PID 14; stack trace: ***
    @     0x7fedfefea280 (unknown)
    @           0xda5e87 rapidjson::GenericValue<>::Accept<>()
    @          0x15fc469 _ZN5doris10JsonReader17_print_json_valueB5cxx11ERKN9rapidjson12GenericValueINS1_4UTF8IcEENS1_19MemoryPoolAllocatorINS1_12CrtAllocatorEEEEE
    @          0x15fd1d9 doris::JsonReader::_write_values_by_jsonpath()
    @          0x15fef2f doris::JsonReader::_handle_flat_array_complex_json()
    @          0x15ff9a1 doris::JsonScanner::get_next()
    @          0x15b6254 doris::BrokerScanNode::scanner_scan()
    @          0x15b6cf1 doris::BrokerScanNode::scanner_worker()
    @          0x26cfccf execute_native_thread_routine
    @     0x7fedfed9fdd5 start_thread
    @     0x7fedff0b1ead __clone
```
- Flink Log
```log
855054648
2021-01-14 17:47:01,157 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - Read timed out		java.net.SocketTimeoutException: Read timed out	 ????
2021-01-14 17:47:01,157 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ?????	http://10.181.83.13:18030/api/ydcdm/dwd_ops_oper_di/_stream_load
2021-01-14 17:47:01,157 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ?1??????
2021-01-14 17:47:01,166 WARN  org.apache.http.client.protocol.ResponseProcessCookies       [] - Invalid cookie header: "set-cookie: PALO_SESSION_ID=e99e9ae0-bce3-4227-93ac-21361f8ded7a; Max-Age=86400; Expires=Fri, 15 Jan 2021 09:47:01 GMT". Invalid 'expires' attribute: Fri, 15 Jan 2021 09:47:01 GMT
2021-01-14 17:47:01,167 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ??????:	http://10.181.83.13:18030/api/ydcdm/dwd_ops_oper_di/_stream_load
2021-01-14 17:47:01,167 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ??????:	{"status":"FAILED","msg":"HTTP method is not allowed: T"}
2021-01-14 17:47:01,168 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - Doris?Sink To Doris Fail?		java.lang.RuntimeException: Doris?Sink To Doris Fail?	 ????
2021-01-14 17:47:01,168 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ?????	http://10.181.83.13:18030/api/ydcdm/dwd_ops_oper_di/_stream_load
2021-01-14 17:47:01,168 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ?2??????
2021-01-14 17:48:01,238 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - Read timed out		java.net.SocketTimeoutException: Read timed out	 ????
2021-01-14 17:48:01,238 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ?????	http://10.181.83.14:18030/api/ydcdm/dwd_ops_oper_di/_stream_load
2021-01-14 17:48:01,238 ERROR com.yd.flink.hand.doris.DorisStreamLoadUtil                  [] - ?3??????
2021-01-14 17:48:01,268 INFO  org.apache.kafka.clients.FetchSessionHandler                 [] - [Consumer client


```

**Comment by HappenLee** [comment]
It's look like load and write json data failed, could you please use gdb to find which json data indeed cause the load failed. Thank you

**Comment by zh0122** [comment]
> It's look like load and write json data failed, could you please use gdb to find which json data indeed cause the load failed. Thank you

Using stream load to put a null list will cause the BE crash. 

```java
// set headers
 // format = "json";  strip_outer_array = "true"  jsonpaths = "XXX"  columns = "XXX" 

        List<Object> list = new ArrayList<>();
        httpPut.setEntity(new StringEntity(list.toString(), Charset.forName("UTF-8")));

        response = httpClient.execute(httpPut);
```

The null list value will cause BE crash.


**Expected behavior**
The BE should return a msg like "json value error" or sth else.

**Comment by zh0122** [comment]
Fixed in #5349
